pkg/resource/k8s-events-config/alloy/events-logger.alloy.template

```diff
@@ -0,0 +1,27 @@
+loki.source.kubernetes_events "local" {
+	namespaces = {{ .ScrapedNamespaces }}
+	forward_to = [loki.write.default.receiver]
+}
+
+// Loki target configuration
+loki.write "default" {
+	endpoint {
+		url                = env("{{ .LokiURLEnvVarName }}")
+		max_backoff_period = "{{ .MaxBackoffPeriod }}"
+		tenant_id          = env("{{ .TenantIDEnvVarName }}")
+
+		basic_auth {
+			username = env("{{ .BasicAuthUsernameEnvVarName }}")
+			password = env("{{ .BasicAuthPasswordEnvVarName }}")
+		}
+	}
+	external_labels = {
+		cluster_id   = "{{ .ClusterID }}",
+		installation = "{{ .Installation }}",
+	}
+}
+
+logging {
+  level  = "info"
```

Comment:
Careful about tabs here,also, you need to add tls config to support installations with private CA as is done for alloy-logs

Reply:
Done.